src/main/kotlin/skytils/skytilsmod/features/impl/handlers/SpamHider.kt

```diff
@@ -411,10 +411,10 @@ class SpamHider {
                     else -> {
                     }
                 }
-            }
+            }<
```

Comment:
Did you accidentally type something here

Reply:
Yeah I didn't mean to already PR earlier apologies for the confusion. Fixed it and was able to test the changes now, should be all good.